app/models/form.rb

```diff
@@ -0,0 +1,4 @@
+class Form < ActiveResource::Base
+  self.site = ENV.fetch('API_BASE')
```

Comment:
I changed this in my branch to be the following so that if we end up needing to handle versioning we can do it per resource - then change the `API_BASE` to no longer include the version

```suggestion
  self.site = "#{ENV.fetch('API_BASE')}/v1"
```

Reply:
Good idea, I've updated it